.gitlab-ci.yml

```diff
@@ -0,0 +1,64 @@
+include:
+  - project: 'city-of-helsinki/kuva/ci-cd-config/ci-configuration'
+    ref: v1
+    file: '/.gitlab-ci-template.yml'
+
+variables:
+  SERVICE_PORT: 80
+  POSTGRES_ENABLED: 0
+
+build-review:
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-review"
+    K8S_SECRET_REACT_APP_API_URI: "https://venepaikka-api.test.hel.ninja/graphql_v2/"
+  only:
+    refs:
+      - branches
+  except:
+    refs:
+      - develop
+      - master
+
+build-staging:
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-staging"
+    K8S_SECRET_REACT_APP_API_URI: "https://venepaikka-api.test.hel.ninja/graphql_v2/"
+  only:
+    refs:
+      - develop
+      - branches
+      - pushes
+
+build-production:
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-production"
+    K8S_SECRET_REACT_APP_API_URI: "https://venepaikka-api.test.hel.ninja/graphql_v2/"
+  only:
+    refs:
+      - master
+
+review:
+  variables:
+    DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-review"
+
+# This will enable staging ci-pipeline
+staging:
+  # By default the staging environment is created from the master-branch.
+  # Here we define that it should be created from the branch called "develop" instead.
+  stage: staging
+  only:
+    refs:
+      - develop
+
+  variables:
+    DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-staging"
+    K8S_SECRET_ALLOWED_HOSTS: "*"
+    K8S_SECRET_DEBUG: 1
+    K8S_SECRET_GCS_BUCKET_NAME: "$KUBE_BUCKET_NAME"
+    K8S_SECRET_SECRET_KEY: "$TF_DJANGO_SECRET_KEY"
+    K8S_SECRET_SKIP_DATABASE_CHECK: 1
+    K8S_SECRET_RESERVATIONS_ADMIN_BASE_URL: "https://venepaikka-admin.test.kuva.hel.ninja"
```

Comment:
These variables are probably not needed by the application.

Reply:
I'm not really sure what any of these variables do... Can I remove this entire section? Is the staging URL specifeid somewhere else?